README.md

```diff
@@ -20,7 +22,10 @@ Innkeeper requires a Postgres DB for operation. For local development, docker ca
 
 To run Innkeeper, execute `sbt run`.
 
-To run the test suite, run `sbt test`.
+## Running the tests
+
+To run the unit test suite, run `sbt test`.
+To run the integration test suite, run `sbt it:test`.
 
```

Comment:
The readme could make it clear, that for a successful it:test, a specific postgres is needed...